functional/FunctionalTest.java

```diff
@@ -1994,11 +2004,22 @@ public static void testGetBucketPolicy(String objectPrefix, PolicyType policyTyp
       if (type != policyType) {
         throw new Exception("[FAILED] Expected: " + policyType + ", Got: " + type);
       }
-      mintSuccessLog("getBucketPolicy(String bucketName, String objectPrefix)", null, startTime);
+      mintSuccessLog("" + policyType + ": getBucketPolicy(String bucketName, String objectPrefix)", null, startTime);
     } catch (Exception e) {
-      mintFailedLog("getBucketPolicy(String bucketName, String objectPrefix)", null, startTime, null,
-                    e.toString() + " >>> " + Arrays.toString(e.getStackTrace()));
-      throw e;
+      ErrorResponse errorResponse = null;
+      if (e instanceof ErrorResponseException) {
+        ErrorResponseException exp = (ErrorResponseException) e;
+        errorResponse = exp.errorResponse();
+      }
+
+      // Ignore NotImplemented error
+      if (errorResponse != null && errorResponse.errorCode() == ErrorCode.NOT_IMPLEMENTED) {
+        mintIgnoredLog("" + policyType + ": getBucketPolicy(String bucketName, String objectPrefix)", null, startTime);
```

Comment:
In mint, `function` is logged as a string with the name of function being tested (getBucketPolicy) in this case. Adding policyType is not helpful.